src/pages/talks/2016-03-building-static-sites-with-react.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import withStyles from 'react-jss';
+
+import Talk from './_talk';
+
+const styles = theme => ({
```

Comment:
None of the pages need styles. Everything will be delegated to the talk component.